easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-2013.5.1-ictce-5.4.0.eb

```diff
@@ -0,0 +1,22 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+##
+name = 'GAMESS-US'
+version = '2013.5.1'
+
+homepage = 'http://www.msg.ameslab.gov/gamess/'
+description = """The General Atomic and Molecular Electronic Structure System (GAMESS)
+is a general ab initio quantum chemistry package."""
+
+toolchain = {'name': 'ictce', 'version': '5.4.0'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'http://www.msg.chem.iastate.edu/GAMESS/download/dist.source.shtml',  # GAMESS-US sources
+]
+sources = [
+    '%(name)s-%(version)s.tar.gz',
```

Comment:
I guess I can. I suppose I thought that SOURCE_TAR_GZ and other macros like it were only useful if the tarball was the sort that could be automatically fetched from the internet and didn't have to be renamed by the user.